src/vs/server/node/remoteAgentEnvironmentImpl.ts

```diff
@@ -356,7 +356,7 @@ export class RemoteAgentEnvironmentChannel implements IServerChannel {
 		return scannedExtensions.map(e => toExtensionDescription(e, false)).filter(ext => {
 			// TODO: remove this once whe decoupled gitpod extensions from gp-code
 			const ignoreExtensions = [
-				'vscode.github-authentication',
+				// 'vscode.github-authentication',
 				'gitpod.gitpod-shared',
 				'gitpod.gitpod-remote-ssh',
 				'gitpod.gitpod-desktop'
```

Comment:
remove `'gitpod.gitpod-desktop'` too please, I forgot to delete it

Reply:
removed 👍

Reply:
you forgot to remove it :smile:

Reply:
I did remove it and push forced